releasenotes/notes/fix-resource-name-when-fastapi-starlette-use-subapps-95ac82b038e6cbf9.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixes root span resource naming when Starlette or Fastapi sub-applications are used.
```

```suggestion
    Fixes root span resource naming when Starlette or Fastapi sub-applications are used.
```

Can we be a bit more descriptive about how they were wrong before and what the new behaviour is. I don't think a typical user would understand what to expect with this change. Maybe an example would help too.

---

Should I use a code example? Or would something like: _"Formerly a call to /sub-app/hello/name would give us a root span resource name of '/sub-app' which wasn't very useful. Now we add the full path '/sub-app/hello/{name}`"_

---

```suggestion
    fastapi/starlette: when using sub-apps, formerly a call to ``/sub-app/hello/{name}`` would give a resource name of ``/sub-app`` which wasn't very useful. Now the full path ``/sub-app/hello/{name}`` is used.
```